vnext/Microsoft.ReactNative/Base/CoreNativeModules.cpp

```diff
@@ -62,9 +61,10 @@ std::vector<facebook::react::NativeModuleDescription> GetCoreModules(
       },
       batchingUIMessageQueue);
 
+  // TODO: Is &&context appropriate if moved in two 'emplace_back' instances?
```

Comment:
For reviewers.

`context` is being moved in two occasions.
Should the first one be replaced by a copy constructor?

Reply:
The first one should be a copy because context is used farther down in the function.

Reply:
Fixed.